pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -35,6 +36,18 @@ func (r *RegistryFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&r.Anon, "registry-anon", false, "Set anonymous auth ($IMGPKG_ANON)")
 
 	cmd.Flags().DurationVar(&r.ResponseHeaderTimeout, "registry-response-header-timeout", 30*time.Second, "Maximum time to allow a request to wait for a server's response headers from the registry (ms|s|m|h)")
+
+	cmd.Flags().SetNormalizeFunc(func(f *pflag.FlagSet, name string) pflag.NormalizedName {
```

Comment:
for the sake of simplifying things, lets not alias original flag to the new one. original flag should not be used by anybody and if it is somehow discovered -- that would just be undefined behaviour from imgpkg's pov.

Reply:
done